### DISCUSSION TITLE
Error Monitoring in tvOS

### DISCUSSION THREAD

**Discussion by ryancw** [react-native-tvos/react-native-tvos#376] [Q&A] [answered]
Hey all, wondering what you're using for error monitoring on the React Native tvOS fork. I tried installing Bugsnag but was unable to get it to work. Any packages working out of the box?

**Comment by douglowder** [comment, marked as answer]
I see that Bugsnag claims to support tvOS: https://docs.bugsnag.com/platforms/tvos/

Can you be more specific about what failed, and why?

I am investigating the possibility of making `react-native-firebase` work, but that is not yet supported.

**Reply by coucoseth** [reply]
hey @douglowder is react-native-firebase currently supported?

**Comment by maxphillipsdev** [comment]
@ryancw Can confirm that bugsnag works but you may need to patch the podspec file to get their SDKs' native code to work with tvos
